Makefile

```diff
@@ -123,6 +125,12 @@ pooled-mixed-mode: cli_integration_tests/CRISPRessoPooled_on_pooled-mixed-mode
 cli_integration_tests/CRISPRessoPooled_on_pooled-mixed-mode: install cli_integration_tests/inputs/Both.Cas9.fastq cli_integration_tests/inputs/ cli_integration_tests/inputs/ cli_integration_tests/inputs/Cas9.amplicons.txt
 	cd cli_integration_tests && cmd="CRISPRessoPooled -r1 inputs/Both.Cas9.fastq -x inputs/small_genome/smallGenome -f inputs/Cas9.amplicons.txt --keep_intermediate --min_reads_to_use_region 100 --debug -n pooled-mixed-mode --place_report_in_output_folder"; $(RUN)
 
+.PHONY: pooled-mixed-mode-align-to-genome
```

Comment:
I think that we can just modify `pooled-mixed-mode` to use this input and delete the `pooled-mixed-mode-align-to-genome` test case. Because `pooled-mixed-mode` wasn't fully testing the mixed mode in the first place. What do you think about this?